Engine/src/Engine/Audio.cpp

```diff
@@ -0,0 +1,69 @@
+#include "Audio.h"
+
+using namespace std;
+using namespace irrklang;
+
+AudioPlayerSingleton* AudioPlayerSingleton::instance;
+
+AudioPlayerSingleton* AudioPlayerSingleton::getInstance()
+{
+	if (instance == 0)
+	{
+		instance = new AudioPlayerSingleton();
+	}
+
+	return instance;
+}
+
+// Plays a sound, takes a path to the sound as a const char, a bool if the sound should loop,
+// a bool if the sound should be initialized paused or play on initialization, and a bool for if
+// sound effects should be used. Sound effects need to be added after using this function using
+// the returned ISound*.
+// More information on sound effects: https://www.ambiera.com/irrklang/docu/index.html#effects
+// 
+ISound* AudioPlayerSingleton::playSound(const char* soundPath, bool loop, bool startPaused, bool useSoundEffects)
+{
+	ISound* theSound = engine->play2D(soundPath, loop, startPaused, true, ESM_AUTO_DETECT, useSoundEffects);
+	if (theSound != 0)
+	{
+		return theSound;
+	}
+	return NULL;
+	//TODO: implement alternate media folder search if multiple media folders
```

Comment:
It's probably better to leave the TODO here and create a task in Trello with the relevant code to be added